index.html

```diff
@@ -26,18 +26,20 @@ <h1>Team SkyBlue</h1>
         </ul>
       </nav>
     </header>
-
-<body>
-  <!-- Main Text -->
-  <section class="main-text">
-    <div class="pic"><img src="images/Yana.jpg" alt="Yana" width="150" height="200"></div>
-    <div class="pic"><img src="images/Miroslav.jpg" alt="Miroslav" width="150" height="200"></div>
-    <div class="pic"><img src="images/Feruz.jpg" alt="Feruz" width="150" height="200"></div>
-    <div class="pic"><img src="images/murielle.png"alt="Murielle" width="150" height="200"></div>
-    <div class="pic"><img src="images/Harun.jpg" alt="Harun" width="150" height="200"></div>
-    <div class="pic"><img src="images/boam.jpg"Boam" width="150" height="200"></div>
-  </section>  
-</body>
```

Comment:
@murvanessa at this stage of solving conflict, you have to keep our parts (that is already on master) which you did correctly. but, you have to keep your part too murielle, dont delete your part. because later we wont have your part on the master when we accept the pull request.